src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -92,8 +92,9 @@ private sealed partial class UIThreadOnly : IDisposable
 
             private readonly IWaitIndicator _waitIndicator;
 
-            public readonly ITextBuffer OutputBuffer;
-            public readonly ITextBuffer StandardInputBuffer;
+            public ITextBuffer OutputBuffer { get; }
+            public ITextBuffer HistoryBuffer { get; private set; }
```

Comment:
`AddInput` affects current user input and can't be used when `SubmitAsync` is in progress.